utils.ts

```diff
@@ -276,3 +276,47 @@ export const cachedGeocoderRequest = async (
 
   return onlineResponse
 }
+
+/**
+ * Checks if a feature collection provides "satisfactory" results for a given queryString.
+ * Satisfactory is defined as having results, having results where at least one is of a set of
+ * preferred layers, and as at least one of the results containing all characters present in the
```

Comment:
I interpreted this as the characters need to be present just in any order, I think it should just say "at least one of the results contains the entirety of the query string"

Reply:
Good idea! Thanks